lab-raziyeh/app/config/router-config.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+module.exports = ['$stateProvider', '$urlRouterProvider',routerConfig ];
+function routerConfig ($stateProvider, $urlRouterProvider){
+  $stateProvider
+
+  .state('home', {
+    url: '/home',
+    template: require('../view/home/home.html'), controller: 'HomeController', controllerAs:'homeCtrl', 
+  })
+
+  .state('signup', {
+    url: '/signup',
+    template: require('../view/signup/signup.html'), controller: 'SignupController',controllerAs:'signupCtrl',
+  })
+
+  .state('gallery', {
+    url: '/gallery',
+    template: require('../view/gallery/gallery.html'), controllerAs:'galleryCtrl',
```

Comment:
@Raziyehbazargan best to drop your controller properties to a new line - for readability